hello-world/test/loop.js

```diff
@@ -0,0 +1,33 @@
+const { calcEthereumTransactionParams } = require("@acala-network/eth-providers");
+
+const txFeePerGas = '199999946752';
+const storageByteDeposit = '100000000000000';
+
+const sleep = async time => new Promise((resolve) => setTimeout(resolve, time));
+
+// this is a helper method to keep feeding tx to node
+// usedful when running node with --instant-sealing
+const loop = async (interval = 2000) => {
+  const ethParams = calcEthereumTransactionParams({
+    gasLimit: '2100001',
+    validUntil: '360001',
+    storageLimit: '64001',
+    txFeePerGas,
+    storageByteDeposit
+  });
+
+  console.log('infinite hello world started!');
```

Comment:
Much like the other PRs, I'd like to reword this a bit to be as user friendly as possible. as this is outwards facing example.

```suggestion
  console.log('Started the infinite HelloWorld deployment loop!');
```